test/testdata

```diff
@@ -155,7 +164,6 @@ MP        QChem       QChemMP4SDQTest       basicQChem4.2         water_mp4sdq.o
 MP        QChem       QChemMP4SDTQTest      basicQChem4.2         water_mp4.out
 
 Polar     DALTON      ReferencePolarTest    basicDALTON-2015      Trp_polar_abalnr.out
-Polar     GAMESS      ReferencePolarTest    basicGAMESS-US2012    Trp_polar_tdhf.out
```

Comment:
Even though the version was wrong, it's meant to be a unit test, so it should be re-run with 2017.